cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -635,7 +635,7 @@ static DataprocConf create(Map<String, String> properties) {
     String imageVersion = getString(properties, IMAGE_VERSION);
     String customImageUri = getString(properties, CUSTOM_IMAGE_URI);
     String gcpCmekKeyName = getString(properties, ENCRYPTION_KEY_NAME);
-    String gcpCmekBucket = getString(properties, "gcsBucket");
+    String gcpCmekBucket = DataprocUtils.getBucketName(getString(properties, "gcsBucket"));
```

Comment:
use the constant `gcsBucket` from `DataprocUtils`. This way for future reference it would be easier to know the places it is used.

Reply:
Done !